src/widget/wtracktableview.cpp

```diff
@@ -340,6 +340,10 @@ void WTrackTableView::slotMouseDoubleClicked(const QModelIndex& index) {
             static_cast<DlgPrefLibrary::TrackDoubleClickAction>(
                     doubleClickActionConfigValue);
 
+    if (doubleClickAction == DlgPrefLibrary::IGNORE) {
```

Comment:
I am afraid Windows has already defined IGNORE. 
That can be fixed by using a enum class, just rename the enum value.

Reply:
I'll rename to DO_NOTHING